test/test_nn.py

```diff
@@ -3931,7 +3931,7 @@ def update_shape(shape, dim, new_dim_size):
         encoder_input_shape = correct_encoder_input_shape
         decoder_input_shape = correct_decoder_input_shape
         wrong_tgt_mask_size = tgt_len + 1
-        test(encoder_input_shape, decoder_input_shape, tgt_mask_len=wrong_tgt_mask_size, raises=True)
+        test(encoder_input_shape, decoder_input_shape, tgt_mask_len=wrong_tgt_mask_size)
```

Comment:
why are we ok with no longer raising for wrong `tgt_mask`s?

Reply:
My last state (which I must say is probably quite out of date by now) was also that we could leave all of the `raises` arguments as they were.

Reply:
There used to be an else-branch on the second if